.github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml

```
@@ -2,26 +2,13 @@ name: 🚀 Production Release To Do List
 description: Used to perform a release (Project maintainers only).
 title: "🚀Production Release"
 body:
-- type: textarea
-  attributes:
-    label: Issues Released
-    placeholder: |
-      Create a list of all the items being released.
-      Example:
-      - #10
-      - #20
-    description: The list of issues that will be released.
-  validations:
-    render: markdown
-
 
 - type: checkboxes
   attributes:
     label: Pre-Release ToDo List
     description: Complete the following items to perform a release.
     options:
-    - label: All issues listed in the **_Issues Released_** section above.
-    - label: All issues in the _**Issues Released**_ section are closed and all issue tasks are complete.
+    - label: All issues in the current milestone are closed and all issue tasks are complete.
```

This should be like the value below.
Just like it was called out in the issue.

I think there was some miscommunication.  Do you think that it is better for having **_current_** instead of **_assigned_**?

```suggestion
    - label: All issues in the assigned milestone are closed and all issue tasks are complete.
```

---

For consistency, I think assigned is good, but is it assigned or chosen?  If it's chosen, then I'd put "current" instead of "assigned."

---

I would say assigned because all of the issues and PRs are literally ASSIGNED to milestones.